tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -23,21 +29,16 @@ module.exports = {
       return util.format(this.elements.permissionCheckbox.selector, permission)
     },
 
-    assertSharingNotAllowed: function () {
+    isSharingAllowed: async function () {
```

Comment:
why not just return boolean value from this method that gives weather sharing is allowed or not. Otherwise change the name as it is misleading. for eg. `getSharePermissionMsg()` or something

Reply:
done!
:rocket: